src/components/StickyList.js

```diff
@@ -0,0 +1,45 @@
+import React, { forwardRef, useEffect, useRef } from 'react';
+import { FixedSizeList as List } from 'react-window';
+
+function StickyList(
+  {
+    follow = false,
+    mode = 'bottom',
+    height,
+    itemCount,
+    itemData,
+    itemSize,
+    width,
+    outerElementType,
+    children,
+  },
+  ref,
+) {
+  const innerRef = useRef();
+  useEffect(() => {
+    if (ref.current && follow && innerRef.current) {
+      if (mode === 'bottom') {
```

Comment:
offsetHeight seems to be a bit off on build preview. You may need to compare offsetHeight with `ref ` height before we do scroll (some condition like below should work)

```js
if (mode === 'bottom' && innerRef.current.offsetHeight > ref.current.props.height)
```

Reply:
hi @sumeesh879. First of all thanks for the comment.
Can you be more specific about the problem? `ref.current.props.height` should always be less than `innerRef.current.offsetHeight`

Reply:
<img width="1424" alt="Screen Shot 2020-06-17 at 8 40 08 AM" src="https://user-images.githubusercontent.com/20029120/84899085-316bcf80-b076-11ea-8ebc-ce084f9016d1.png">


Sorry, I was checking out the netilfy preview and saw list is being cut off at the top. 
On initial render, something like above is happening. I thought it would be something to do with innerRef and outerRef/ref height.

Reply:
@marcosvega91, I can reproduce that, but only on the first load.

1. go to https://testing-playground.com
2. now, edit the URL, and just add the `/events` part, don't remove anything.
3. hit <kbd>enter</kbd>
4. see this:
    ![image](https://user-images.githubusercontent.com/1196524/84900873-fd5ed180-b0aa-11ea-8b41-d75c3fe72759.png)

Reply:
Now should be good 😄  👍. Thanks @sumeesh879 for reporting